htdocs/core/class/notify.class.php

```diff
@@ -1105,6 +1113,13 @@ public function send($notifcode, $object, $filename_list = array(), $mimetype_li
 						$object_type = 'ficheinter';
 						$mesg = $langs->transnoentitiesnoconv("EMailTextInterventionValidated", $link);
 						break;
+					case 'FICHINTER_MODIFY':
+						$link = '<a href="'.$urlwithroot.'/fichinter/card.php?id='.$object->id.'&entity='.$object->entity.'">'.$newref.'</a>';
+						$context_info = array_key_exists('signature', $object->context) ? $object->getLibSignedStatus() : '';
+						$dir_output = $conf->ficheinter->dir_output;
```

Comment:
@hregis Recommends multidir_output

Reply:
Changing `$dir_output` to 
`$dir_output = $conf->ficheinter->multidir_output[$object->entity]."/".get_exdir(0, 0, 0, 1, $object);`
doesn't directly work. The PDF is no longer attached to the email notification. Maybe my code is not quite right or maybe this needs refactoring elsewhere to work ?

It seems other notifications could be refactored to make use of `multidir_output` as well. As per contributions guidelines, I think it would be best not to overload this PR, and rework your recommendation in another PR. 

Thank you for input.

Reply:
@hregis is in a better position than me to recommend the details of the how-to.  What I understood from his feedback in another PR was that a simple replacement as you did should work.
Best guess is that the entity property is not set, I've seen some code where a test is made to see if it is set and use 1 if it is not.

Reply:
For information, multidir_output as introduce to be able to manage multicompany (1 directory for each company).
But we have now a function called getMultidirOutput() thatmust be used to replace any call of ->dir_output and ->multidir_output.
The function will return the multidir_output with more check on compatibilty.

Reply:
Thanks will keep this in mind for future PR